### DISCUSSION TITLE
issue with install

### DISCUSSION THREAD

**Discussion by Ivcannons00420** [oobabooga/text-generation-webui#924] [Q&A]
![issue](https://user-images.githubusercontent.com/96838966/230737514-90fe5bb8-4fcf-4cf3-af72-419dc25ffb52.jpg)

**Comment by Ivcannons00420** [comment]
__
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/

pytorch/noarch                                      10.0kB @  65.0kB/s  0.2s
pytorch/win-64                                     119.6kB @ 740.8kB/s  0.2s
nvidia/win-64                                       89.0kB @ 512.5kB/s  0.2s
nvidia/noarch                                        2.8kB @  14.3kB/s  0.0s
nvidia/label/cuda-11.7.0/win-64                      5.4kB @  26.4kB/s  0.2s
nvidia/label/cuda-11.7.0/noarch                    130.0 B @ 598.0 B/s  0.2s
conda-forge/noarch                                  11.8MB @   4.9MB/s  2.8s
conda-forge/win-64                                  19.8MB @   4.5MB/s  5.3s

Transaction

  Prefix: C:\Program Files\oobabooga\oobabooga-windows (1)\oobabooga-windows\installer_files\env

  Updating specs:

   - python=3.10.9
   - pytorch==2[build=py3.10_cuda11.7*]
   - torchvision
   - torchaudio
   - pytorch-cuda=11.7
   - cuda-toolkit
   - ninja
   - git


  Package                               Version  Build                     Channel                              Size
----------------------------------------------------------------------------------------------------------------------
  Install:
----------------------------------------------------------------------------------------------------------------------

  + blas                                  2.116  mkl                       conda-forge/win-64                   14kB
  + blas-devel                            3.9.0  16_win64_mkl              conda-forge/win-64                   13kB
  + brotlipy                              0.7.0  py310h8d17308_1005        conda-forge/win-64                  340kB
  + bzip2                                 1.0.8  h8ffe710_4                conda-forge/win-64                  152kB
  + ca-certificates                   2022.12.7  h5b45459_0                conda-forge/win-64                  147kB
  + certifi                           2022.12.7  pyhd8ed1ab_0              conda-forge/noarch                  151kB
  + cffi                                 1.15.1  py310h628cb3f_3           conda-forge/win-64                  234kB
  + charset-normalizer                    3.1.0  pyhd8ed1ab_0              conda-forge/noarch                   45kB
  + cryptography                         40.0.1  py310h6e82f81_0           conda-forge/win-64                    1MB
  + cuda-cccl                           11.7.58  0                         nvidia/label/cuda-11.7.0/win-64       1MB
  + cuda-command-line-tools              11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-compiler                        11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-cudart                         11.7.60  0                         nvidia/label/cuda-11.7.0/win-64       1MB
  + cuda-cudart-dev                     11.7.60  0                         nvidia/label/cuda-11.7.0/win-64     712kB
  + cuda-cuobjdump                      11.7.50  0                         nvidia/label/cuda-11.7.0/win-64       3MB
  + cuda-cupti                          11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      11MB
  + cuda-cuxxfilt                       11.7.50  0                         nvidia/label/cuda-11.7.0/win-64     169kB
  + cuda-documentation                  11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      93kB
  + cuda-libraries                       11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-libraries-dev                   11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-memcheck                       11.7.50  0                         nvidia/label/cuda-11.7.0/win-64     187kB
  + cuda-nsight-compute                  11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-nvcc                           11.7.64  0                         nvidia/label/cuda-11.7.0/win-64      46MB
  + cuda-nvdisasm                       11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      33MB
  + cuda-nvml-dev                       11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      87kB
  + cuda-nvprof                         11.7.50  0                         nvidia/label/cuda-11.7.0/win-64       2MB
  + cuda-nvprune                        11.7.50  0                         nvidia/label/cuda-11.7.0/win-64     155kB
  + cuda-nvrtc                          11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      75MB
  + cuda-nvrtc-dev                      11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      15MB
  + cuda-nvtx                           11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      44kB
  + cuda-nvvp                           11.7.50  0                         nvidia/label/cuda-11.7.0/win-64     119MB
  + cuda-runtime                         11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-sanitizer-api                  11.7.50  0                         nvidia/label/cuda-11.7.0/win-64      13MB
  + cuda-toolkit                         11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-tools                           11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + cuda-visual-tools                    11.7.0  0                         nvidia/label/cuda-11.7.0/win-64       1kB
  + filelock                             3.11.0  pyhd8ed1ab_0              conda-forge/noarch                   14kB
  + freetype                             2.12.1  h546665d_1                conda-forge/win-64                  497kB
  + git                                  2.40.0  h57928b3_1                conda-forge/win-64                   98MB
  + idna                                    3.4  pyhd8ed1ab_0              conda-forge/noarch                   57kB
  + intel-openmp                       2023.0.0  h57928b3_25922            conda-forge/win-64                    2MB
  + jinja2                                3.1.2  pyhd8ed1ab_1              conda-forge/noarch                  101kB
  + jpeg                                     9e  hcfcfb64_3                conda-forge/win-64                  289kB
  + lcms2                                  2.15  ha5c8aab_0                conda-forge/win-64                  498kB
  + lerc                                  4.0.0  h63175ca_0                conda-forge/win-64                  194kB
  + libblas                               3.9.0  16_win64_mkl              conda-forge/win-64                    6MB
  + libcblas                              3.9.0  16_win64_mkl              conda-forge/win-64                    6MB
  + libcublas                        11.10.1.25  0                         nvidia/label/cuda-11.7.0/win-64      24kB
  + libcublas-dev                    11.10.1.25  0                         nvidia/label/cuda-11.7.0/win-64     300MB
  + libcufft                          10.7.2.50  0                         nvidia/label/cuda-11.7.0/win-64       6kB
  + libcufft-dev                      10.7.2.50  0                         nvidia/label/cuda-11.7.0/win-64     262MB
  + libcurand                        10.2.10.50  0                         nvidia/label/cuda-11.7.0/win-64       3kB
  + libcurand-dev                    10.2.10.50  0                         nvidia/label/cuda-11.7.0/win-64      52MB
  + libcusolver                       11.3.5.50  0                         nvidia/label/cuda-11.7.0/win-64      30kB
  + libcusolver-dev                   11.3.5.50  0                         nvidia/label/cuda-11.7.0/win-64      91MB
  + libcusparse                       11.7.3.50  0                         nvidia/label/cuda-11.7.0/win-64      14kB
  + libcusparse-dev                   11.7.3.50  0                         nvidia/label/cuda-11.7.0/win-64     155MB
  + libdeflate                             1.17  hcfcfb64_0                conda-forge/win-64                  152kB
  + libffi                                3.4.2  h8ffe710_5                conda-forge/win-64                   42kB
  + libhwloc                              2.9.0  h51c2c0f_0                conda-forge/win-64                    3MB
  + libiconv                               1.17  h8ffe710_0                conda-forge/win-64                  715kB
  + liblapack                             3.9.0  16_win64_mkl              conda-forge/win-64                    6MB
  + liblapacke                            3.9.0  16_win64_mkl              conda-forge/win-64                    6MB
  + libnpp                            11.7.3.21  0                         nvidia/label/cuda-11.7.0/win-64     293kB
  + libnpp-dev                        11.7.3.21  0                         nvidia/label/cuda-11.7.0/win-64     120MB
  + libnvjpeg                         11.7.2.34  0                         nvidia/label/cuda-11.7.0/win-64       4kB
  + libnvjpeg-dev                     11.7.2.34  0                         nvidia/label/cuda-11.7.0/win-64       2MB
  + libpng                               1.6.39  h19919ed_0                conda-forge/win-64                  344kB
  + libsqlite                            3.40.0  hcfcfb64_0                conda-forge/win-64                  670kB
  + libtiff                               4.5.0  hf8721a0_2                conda-forge/win-64                    1MB
  + libuv                                1.44.2  h8ffe710_0                conda-forge/win-64                  370kB
  + libwebp-base                          1.3.0  hcfcfb64_0                conda-forge/win-64                  268kB
  + libxcb                                 1.13  hcd874cb_1004             conda-forge/win-64                    1MB
  + libxml2                              2.10.3  hc3477c8_6                conda-forge/win-64                    2MB
  + libzlib                              1.2.13  hcfcfb64_4                conda-forge/win-64                   71kB
  + m2w64-gcc-libgfortran                 5.3.0  6                         conda-forge/win-64                  351kB
  + m2w64-gcc-libs                        5.3.0  7                         conda-forge/win-64                  532kB
  + m2w64-gcc-libs-core                   5.3.0  7                         conda-forge/win-64                  219kB
  + m2w64-gmp                             6.1.0  2                         conda-forge/win-64                  744kB
  + m2w64-libwinpthread-git  5.0.0.4634.697f757  2                         conda-forge/win-64                   32kB
  + markupsafe                            2.1.2  py310h8d17308_0           conda-forge/win-64                   26kB
  + mkl                                2022.1.0  h6a75c08_874              conda-forge/win-64                  192MB
  + mkl-devel                          2022.1.0  h57928b3_875              conda-forge/win-64                    7MB
  + mkl-include                        2022.1.0  h6a75c08_874              conda-forge/win-64                  779kB
  + mpmath                                1.3.0  pyhd8ed1ab_0              conda-forge/noarch                  438kB
  + msys2-conda-epoch                  20160418  1                         conda-forge/win-64                    3kB
  + networkx                                3.1  pyhd8ed1ab_0              conda-forge/noarch                    1MB
  + ninja                                1.11.1  h91493d7_0                conda-forge/win-64                  279kB
  + nsight-compute                  2022.2.0.13  0                         nvidia/label/cuda-11.7.0/win-64     352MB
  + numpy                                1.24.2  py310hd02465a_0           conda-forge/win-64                    6MB
  + openjpeg                              2.5.0  ha2aaf27_2                conda-forge/win-64                  237kB
  + openssl                               3.1.0  hcfcfb64_0                conda-forge/win-64                    7MB
  + pillow                                9.4.0  py310hdbb7713_1           conda-forge/win-64                   46MB
  + pip                                  23.0.1  pyhd8ed1ab_0              conda-forge/noarch                    1MB
  + pthread-stubs                           0.4  hcd874cb_1001             conda-forge/win-64                    6kB
  + pthreads-win32                        2.9.1  hfa6e2cd_3                conda-forge/win-64                  144kB
  + pycparser                              2.21  pyhd8ed1ab_0              conda-forge/noarch                  103kB
  + pyopenssl                            23.1.1  pyhd8ed1ab_0              conda-forge/noarch                  128kB
  + pysocks                               1.7.1  pyh0701188_6              conda-forge/noarch                   19kB
  + python                               3.10.9  h4de0772_0_cpython        conda-forge/win-64                   16MB
  + python_abi                             3.10  3_cp310                   conda-forge/win-64                    6kB
  + pytorch                               2.0.0  py3.10_cuda11.7_cudnn8_0  pytorch/win-64                        1GB
  + pytorch-cuda                           11.7  h16d0643_3                pytorch/win-64                        7kB
  + pytorch-mutex                           1.0  cuda                      pytorch/noarch                        3kB
  + requests                             2.28.2  pyhd8ed1ab_1              conda-forge/noarch                   57kB
  + setuptools                           67.6.1  pyhd8ed1ab_0              conda-forge/noarch                  580kB
  + sympy                                1.11.1  pyh04b8f61_3              conda-forge/noarch                    5MB
  + tbb                                2021.8.0  h91493d7_0                conda-forge/win-64                  154kB
  + tk                                   8.6.12  h8ffe710_0                conda-forge/win-64                    4MB
  + torchaudio                            2.0.0  py310_cu117               pytorch/win-64                        6MB
  + torchvision                          0.15.0  py310_cu117               pytorch/win-64                        8MB
  + typing_extensions                     4.5.0  pyha770c72_0              conda-forge/noarch                   31kB
  + tzdata                                2023c  h71feb2d_0                conda-forge/noarch                  118kB
  + ucrt                           10.0.22621.0  h57928b3_0                conda-forge/win-64                    1MB
  + urllib3                             1.26.15  pyhd8ed1ab_0              conda-forge/noarch                  113kB
  + vc                                     14.3  hb6edc58_10               conda-forge/win-64                   16kB
  + vs2015_runtime                  14.34.31931  h4c5c07a_10               conda-forge/win-64                  725kB
  + wheel                                0.40.0  pyhd8ed1ab_0              conda-forge/noarch                   56kB
  + win_inet_pton                         1.1.0  pyhd8ed1ab_6              conda-forge/noarch                    8kB
  + xorg-libxau                           1.0.9  hcd874cb_0                conda-forge/win-64                   58kB
  + xorg-libxdmcp                         1.1.3  hcd874cb_0                conda-forge/win-64                   68kB
  + xz                                    5.2.6  h8d14728_0                conda-forge/win-64                  218kB
  + zstd                                  1.5.2  h12be248_6                conda-forge/win-64                  288kB

  Summary:

  Install: 123 packages

  Total download: 3GB

----------------------------------------------------------------------------------------------------------------------



Transaction starting
pytorch-mutex                                        2.9kB @  25.0kB/s  0.1s
msys2-conda-epoch                                    3.2kB @  26.2kB/s  0.1s
ca-certificates                                    146.6kB @   1.1MB/s  0.1s
lerc                                               194.4kB @   1.1MB/s  0.0s
pthreads-win32                                     144.3kB @ 711.4kB/s  0.1s
m2w64-gmp                                          743.5kB @   3.2MB/s  0.1s
ucrt                                                 1.3MB @   5.3MB/s  0.2s
m2w64-gcc-libgfortran                              350.7kB @   1.1MB/s  0.1s
ninja                                              279.2kB @ 810.6kB/s  0.1s
libpng                                             343.9kB @ 827.6kB/s  0.1s
xorg-libxau                                         58.0kB @ 122.4kB/s  0.1s
openjpeg                                           237.1kB @ 388.3kB/s  0.1s
libtiff                                              1.2MB @   1.8MB/s  0.3s
tk                                                   3.7MB @   4.6MB/s  0.6s
blas-devel                                          13.5kB @  15.8kB/s  0.1s
openssl                                              7.5MB @   8.3MB/s  0.7s
cuda-nvml-dev                                       87.4kB @  73.7kB/s  0.3s
cuda-documentation                                  93.4kB @  78.5kB/s  0.3s
liblapack                                            5.9MB @   3.8MB/s  0.9s
cuda-nvtx                                           43.9kB @  26.6kB/s  0.5s
libcublas                                           24.5kB @  14.7kB/s  0.5s
cuda-command-line-tools                              1.4kB @ 657.0 B/s  0.4s
git                                                 97.8MB @  34.4MB/s  2.8s
idna                                                56.7kB @  19.7kB/s  0.0s
pip                                                  1.4MB @ 469.2kB/s  0.8s
typing_extensions                                   31.3kB @  10.7kB/s  0.0s
pysocks                                             19.3kB @   6.6kB/s  0.0s
cryptography                                         1.1MB @ 347.0kB/s  0.1s
pyopenssl                                          128.1kB @  40.0kB/s  0.1s
pillow                                              46.2MB @   9.5MB/s  2.0s
mkl-include                                        778.6kB @ 156.9kB/s  0.1s
vs2015_runtime                                     725.0kB @ 144.1kB/s  0.1s
m2w64-gcc-libs-core                                219.2kB @  43.2kB/s  0.0s
libwebp-base                                       268.4kB @  52.2kB/s  0.1s
bzip2                                              152.2kB @  29.4kB/s  0.0s
libzlib                                             71.2kB @  13.6kB/s  0.0s
zstd                                               288.4kB @  54.8kB/s  0.0s
freetype                                           497.4kB @  93.3kB/s  0.1s
tbb                                                153.6kB @  28.5kB/s  0.1s
mkl-devel                                            7.5MB @   1.3MB/s  0.6s
liblapacke                                           5.9MB @ 914.5kB/s  0.5s
mkl                                                191.6MB @  22.9MB/s  7.8s
pytorch                                              1.3GB @  45.8MB/s 24.1s
cuda-memcheck                                      187.0kB @   6.8kB/s  0.2s
cuda-cccl                                            1.2MB @  44.3kB/s  0.4s
libnvjpeg-dev                                        1.9MB @  67.2kB/s  0.5s
cuda-visual-tools                                    1.4kB @  47.0 B/s  0.3s
pycparser                                          102.7kB @   3.5kB/s  0.2s
mpmath                                             438.3kB @  15.1kB/s  0.1s
sympy                                                4.8MB @ 163.1kB/s  0.2s
cffi                                               233.7kB @   8.0kB/s  0.0s
urllib3                                            112.5kB @   3.8kB/s  0.0s
torchaudio                                           5.9MB @ 201.3kB/s  0.2s
m2w64-libwinpthread-git                             31.9kB @   1.1kB/s  0.1s
libdeflate                                         151.9kB @   5.1kB/s  0.0s
libuv                                              370.3kB @  12.5kB/s  0.1s
libxml2                                              1.6MB @  53.7kB/s  0.1s
xorg-libxdmcp                                       67.9kB @   2.3kB/s  0.0s
libxcb                                               1.3MB @  44.9kB/s  0.1s
blas                                                14.1kB @ 470.0 B/s  0.0s
libcusparse                                         13.8kB @ 457.0 B/s  0.2s
libnvjpeg                                            4.4kB @ 144.0 B/s  0.3s
cuda-nvprune                                       155.2kB @   5.1kB/s  0.3s
cuda-libraries                                       1.4kB @  45.0 B/s  0.3s
cuda-toolkit                                         1.4kB @  43.0 B/s  0.3s
certifi                                            150.8kB @   4.8kB/s  0.2s
numpy                                                5.9MB @ 186.8kB/s  0.2s
requests                                            56.9kB @   1.8kB/s  0.0s
cuda-nvcc                                           46.3MB @   1.5MB/s 25.3s
python_abi                                           6.1kB @ 193.0 B/s  0.0s
libsqlite                                          669.9kB @  21.1kB/s  0.0s
pthread-stubs                                        6.4kB @ 201.0 B/s  0.0s
tzdata                                             117.6kB @   3.7kB/s  0.0s
jpeg                                               289.4kB @   9.1kB/s  0.2s
libcurand                                            3.1kB @  97.0 B/s  0.2s
libcusolver                                         30.0kB @ 935.0 B/s  0.2s
cuda-nvrtc-dev                                      15.0MB @ 449.4kB/s  1.4s
win_inet_pton                                        8.2kB @ 243.0 B/s  0.1s
markupsafe                                          25.9kB @ 771.0 B/s  0.0s
torchvision                                          8.1MB @ 238.5kB/s  0.3s
libiconv                                           714.5kB @  21.0kB/s  0.0s
m2w64-gcc-libs                                     532.4kB @  15.6kB/s  0.1s
libblas                                              5.9MB @ 172.0kB/s  0.2s
libnpp                                             292.8kB @   8.5kB/s  0.2s
cuda-cupti                                          10.6MB @ 300.1kB/s  1.0s
cuda-nvrtc                                          75.4MB @   2.0MB/s 29.7s
pytorch-cuda                                         6.9kB @ 180.0 B/s  0.2s
networkx                                             1.5MB @  38.0kB/s  0.2s
jinja2                                             101.4kB @   2.6kB/s  0.1s
libffi                                              42.1kB @   1.1kB/s  0.0s
lcms2                                              498.4kB @  12.9kB/s  0.1s
libcufft                                             5.6kB @ 145.0 B/s  0.2s
cuda-cuobjdump                                       2.6MB @  65.9kB/s  0.6s
cuda-runtime                                         1.3kB @  33.0 B/s  0.3s
cuda-nvvp                                          119.1MB @   3.0MB/s  4.2s
filelock                                            14.1kB @ 355.0 B/s  0.1s
vc                                                  15.7kB @ 393.0 B/s  0.1s
cuda-sanitizer-api                                  13.2MB @ 322.2kB/s  1.3s
libcusolver-dev                                     90.8MB @   2.2MB/s  9.2s
cuda-tools                                           1.3kB @  32.0 B/s  0.3s
intel-openmp                                         2.2MB @  53.5kB/s  0.3s
cuda-cudart                                          1.4MB @  34.0kB/s  0.5s
charset-normalizer                                  44.9kB @   1.1kB/s  0.2s
libcblas                                             5.9MB @ 139.3kB/s  0.3s
cuda-cuxxfilt                                      169.2kB @   4.0kB/s  0.2s
libnpp-dev                                         119.9MB @   2.8MB/s 41.0s
setuptools                                         580.3kB @  13.6kB/s  0.2s
cuda-libraries-dev                                   1.4kB @  32.0 B/s  0.3s
brotlipy                                           339.9kB @   7.9kB/s  0.2s
cuda-nsight-compute                                  1.3kB @  30.0 B/s  0.2s
xz                                                 217.8kB @   5.0kB/s  0.1s
libcurand-dev                                       51.5MB @   1.0MB/s  8.0s
cuda-nvprof                                          1.6MB @  32.3kB/s  0.5s
cuda-nvdisasm                                       33.2MB @ 632.0kB/s  2.4s
libcublas-dev                                      300.3MB @   5.7MB/s 51.1s
wheel                                               55.7kB @   1.1kB/s  0.1s
cuda-cudart-dev                                    712.0kB @  13.5kB/s  0.4s
cuda-compiler                                        1.3kB @  25.0 B/s  0.2s
libhwloc                                             2.5MB @  47.4kB/s  0.4s
python                                              15.9MB @ 297.1kB/s  0.6s
libcufft-dev                                       261.9MB @   4.5MB/s 14.8s
libcusparse-dev                                    154.8MB @   2.6MB/s 16.2s
nsight-compute                                     352.5MB @   5.9MB/s 20.4s
Linking pytorch-mutex-1.0-cuda
Linking git-2.40.0-h57928b3_1
Linking ca-certificates-2022.12.7-h5b45459_0
Linking ucrt-10.0.22621.0-h57928b3_0
Linking msys2-conda-epoch-20160418-1
Linking mkl-include-2022.1.0-h6a75c08_874
Linking intel-openmp-2023.0.0-h57928b3_25922
Linking python_abi-3.10-3_cp310
Linking vs2015_runtime-14.34.31931-h4c5c07a_10
Linking m2w64-libwinpthread-git-5.0.0.4634.697f757-2
Linking m2w64-gmp-6.1.0-2
Linking vc-14.3-hb6edc58_10
Linking m2w64-gcc-libs-core-5.3.0-7
Linking pthreads-win32-2.9.1-hfa6e2cd_3
Linking libiconv-1.17-h8ffe710_0
Linking libdeflate-1.17-hcfcfb64_0
Linking lerc-4.0.0-h63175ca_0
Linking libwebp-base-1.3.0-hcfcfb64_0
Linking jpeg-9e-hcfcfb64_3
Linking openssl-3.1.0-hcfcfb64_0
Linking libffi-3.4.2-h8ffe710_5
Linking bzip2-1.0.8-h8ffe710_4
Linking ninja-1.11.1-h91493d7_0
Linking libuv-1.44.2-h8ffe710_0
Linking xz-5.2.6-h8d14728_0
Linking tk-8.6.12-h8ffe710_0
Linking libzlib-1.2.13-hcfcfb64_4
Linking libsqlite-3.40.0-hcfcfb64_0
Linking m2w64-gcc-libgfortran-5.3.0-6
Linking libxml2-2.10.3-hc3477c8_6
Linking zstd-1.5.2-h12be248_6
Linking libpng-1.6.39-h19919ed_0
Linking m2w64-gcc-libs-5.3.0-7
Linking libhwloc-2.9.0-h51c2c0f_0
Linking libtiff-4.5.0-hf8721a0_2
Linking freetype-2.12.1-h546665d_1
Linking xorg-libxdmcp-1.1.3-hcd874cb_0
Linking xorg-libxau-1.0.9-hcd874cb_0
Linking pthread-stubs-0.4-hcd874cb_1001
Linking tbb-2021.8.0-h91493d7_0
Linking openjpeg-2.5.0-ha2aaf27_2
Linking lcms2-2.15-ha5c8aab_0
Linking libxcb-1.13-hcd874cb_1004
Linking mkl-2022.1.0-h6a75c08_874
Linking mkl-devel-2022.1.0-h57928b3_875
Linking libblas-3.9.0-16_win64_mkl
Linking liblapack-3.9.0-16_win64_mkl
Linking libcblas-3.9.0-16_win64_mkl
Linking liblapacke-3.9.0-16_win64_mkl
Linking blas-devel-3.9.0-16_win64_mkl
Linking blas-2.116-mkl
Linking tzdata-2023c-h71feb2d_0
Linking cuda-documentation-11.7.50-0
Linking cuda-nvcc-11.7.64-0
warning  libmamba [cuda-nvcc-11.7.64-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking nsight-compute-2022.2.0.13-0
warning  libmamba [nsight-compute-2022.2.0.13-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-nvml-dev-11.7.50-0
warning  libmamba [cuda-nvml-dev-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking libcusparse-11.7.3.50-0
Linking libnpp-11.7.3.21-0
Linking libcusolver-11.3.5.50-0
Linking libcufft-10.7.2.50-0
Linking cuda-nvrtc-11.7.50-0
Linking cuda-cudart-11.7.60-0
Linking libcurand-10.2.10.50-0
Linking libnvjpeg-11.7.2.34-0
Linking libcublas-11.10.1.25-0
Linking cuda-nvprof-11.7.50-0
warning  libmamba [cuda-nvprof-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-nvdisasm-11.7.50-0
warning  libmamba [cuda-nvdisasm-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-nvtx-11.7.50-0
warning  libmamba [cuda-nvtx-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-memcheck-11.7.50-0
warning  libmamba [cuda-memcheck-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-sanitizer-api-11.7.50-0
warning  libmamba [cuda-sanitizer-api-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-cupti-11.7.50-0
warning  libmamba [cuda-cupti-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-cuxxfilt-11.7.50-0
warning  libmamba [cuda-cuxxfilt-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-cuobjdump-11.7.50-0
warning  libmamba [cuda-cuobjdump-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-nvprune-11.7.50-0
warning  libmamba [cuda-nvprune-11.7.50-0] The following files were already present in the environment:
    - LICENSE
    - build_env_setup.bat
    - conda_build.bat
    - metadata_conda_debug.yaml
Linking cuda-cccl-11.7.58-0
Linking cuda-nsight-compute-11.7.0-0
Linking libcusparse-dev-11.7.3.50-0
Linking libnpp-dev-11.7.3.21-0
Linking libcusolver-dev-11.3.5.50-0
Linking libcufft-dev-10.7.2.50-0
Linking cuda-nvrtc-dev-11.7.50-0
Linking libcurand-dev-10.2.10.50-0
Linking libnvjpeg-dev-11.7.2.34-0
Linking libcublas-dev-11.10.1.25-0
Linking cuda-libraries-11.7.0-0
Linking cuda-nvvp-11.7.50-0
warning  libmamba Invalid package cache, file 'C:\Program Files\oobabooga\oobabooga-windows (1)\oobabooga-windows\instagerror    libmamba Cannot find a valid extracted directory cache for 'cuda-nvvp-11.7.50-0.tar.bz2'
critical libmamba Package cache error.

Conda environment creation failed.
Press any key to continue . . .

**Comment by khot-aditya** [comment]
I have same issue "Conda environment creation failed.". Someone please help with this.

**Reply by khot-aditya** [reply]
When i moved the folder into c: drive with shorter path, it worked for me. But I ran into another issue the output of Assistant is gibberish. No matter what prompt I write it replies me with gibberish words. I am trying to find solution.

**Comment by Kepler-16b** [comment]
delete the env folder under 
oobabooga-windows\installer_files
and try again? I got it working after a couple of trys.

**Reply by Dinosoreo32** [reply]
Tried it just now and received an ( recursive_directory_iterator::recursive_directory_iterator: The directory name is invalid.: "C:\Users")

**Reply by Dinosoreo32** [reply]
it seems like there are 2 pesky files that aren't caching correctly that throw off the entire program

**Reply by Squeezitgirdle** [reply]
> delete the env folder under oobabooga-windows\installer_files and try again? I got it working after a couple of trys.

You are a wonderful person. This was my issue. Thank you. Dunno why it failed the first time but it's good now.

**Comment by agplusman** [comment]
I get the same error for NVIDIA install A)

error libmamba Cannot find a valid extracted directory cache for 'cuda-nvvp-11.7.50-0.tar.bz2' critical libmamba Package cache error.

and its also mentioned here https://www.reddit.com/r/Oobabooga/comments/125e6it/the_windows_oneclick_installer_has_been_updated/

**Comment by joshuawegner** [comment]
Solved it by double checking the directory path. You cant have spaces and special characters in it. Without spaces or special characters the installation works fine 👍
